phpBB/styles/prosilver/theme/forms.css

```diff
@@ -281,13 +281,19 @@ fieldset.submit-buttons input {
 /* Emoticons panel */
 .smiley-box {
 	float: right;
-	width: 18%;
+	width: 100%;
 }
```

Comment:
please remove that comment as its not emoticons lets keep all the smiley stuff together in the same file and same location.